bundles/org.openhab.binding.caddx/src/test/java/org/openhab/binding/caddx/internal/CaddxMessageReaderUtil.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.caddx.internal;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.util.HexUtils;
+
+/**
+ * Util class to read test input messages.
+ *
+ * @author Georgios Moutsos - Initial contribution
+ */
+@NonNullByDefault
+public final class CaddxMessageReaderUtil {
+    private static final String MESSAGE_EXT = ".msg";
+
+    private CaddxMessageReaderUtil() {
+        // Util class
+    }
+
+    /**
+     * Reads the raw bytes of the message given the file relative to this package and returns the objects.
+     *
+     * @param messageName name of the telegram file to read
+     * @return The raw bytes of a telegram
+     */
+    public static byte[] readRawMessage(String messageName) {
+        try (InputStream is = CaddxMessageReaderUtil.class.getResourceAsStream(messageName + MESSAGE_EXT)) {
+            String hexString = new BufferedReader(new InputStreamReader(is)).lines().collect(Collectors.joining("\n"));
```

Comment:
The `BufferedReader` and `InputStreamReader` should also be part of the `try`. Can you file a follow-up PR, as the 3.x PR is already merged?

Reply:
yes will do.